torch/_torch_docs.py

```diff
@@ -772,6 +775,18 @@ def parse_kwargs(desc):
              -1.0969, -0.4614],
             [-0.1034, -0.5790,  0.1497, -0.1034, -0.5790,  0.1497, -0.1034,
              -0.5790,  0.1497]])
+    >>> x = torch.ones(2, 3, device='cpu')
+    >>> y = torch.zeros(1, 4, device='cpu')
+    >>> torch.cat([x, y], dim=0, pad=True, pad_value=0.5)
+    tensor([[1.0000, 1.0000, 1.0000, 0.5000],
+            [1.0000, 1.0000, 1.0000, 0.5000],
+            [0.0000, 0.0000, 0.0000, 0.0000]])
+    >>> x = torch.ones(2, 3, device='cuda')
+    >>> y = torch.zeros(1, 4, device='cuda')
+    >>> torch.cat([x, y], dim=0, pad=True, pad_value=0.5)
+    tensor([[1.0000, 1.0000, 1.0000, 0.5000],
+            [1.0000, 1.0000, 1.0000, 0.5000],
+            [0.0000, 0.0000, 0.0000, 0.0000]])
```

Comment:
wait -- that doesn't create a cuda tensor?  (You don't need this example since it's redundant, but it's weird that it's not a cuda tensor).